### DISCUSSION TITLE
knx device

### DISCUSSION THREAD

**Discussion by florinkurky** [Supergiovane/node-red-contrib-knx-ultimate#403] [Q&A]
hy. i have a ls100100 and y want to use with node-red and alexa. i configured all but dosen't work. node-red dosen't communicate with knx . in cmd i have this error " KNX-Unsecure: connection to inseucre interface/router using node KNX Gateway.

**Comment by Supergiovane** [comment]
Hi
Please follow the indications in the issue report, as i need such infos.
